src/Swarm/Game/State.hs

```diff
@@ -414,10 +433,63 @@ addRobot r = do
     %= M.insertWith IS.union (r ^. robotLocation) (IS.singleton rid)
   internalActiveRobots %= IS.insert rid
 
--- | Create an initial game state record for a particular game type,
---   first loading entities and recipies from disk.
-initGameState :: Maybe Seed -> Maybe String -> Maybe String -> ExceptT Text IO GameState
-initGameState cmdlineSeed scenarioToLoad toRun = do
+maxMessageQueueSize :: Int
```

Comment:
I didn't change anything from here down to `Initialization`, I just moved it to try to organize the code a bit better.